board/app/models/user.php

```diff
@@ -135,5 +132,143 @@ public function is_email_exist()
         $username_exist = $db->row("SELECT email FROM user where email = ?", array($this->email));
         return (!$username_exist);
     }
-}
-?>
\ No newline at end of file
+
+
+    public static function get($user_id) {
+        $db = DB::conn();
+        $row = $db->row('SELECT * FROM user WHERE id = ?', array($user_id));
+        return $row;
+    }   
+
+    public static function getAll($id) {
+        $user = array();
+        $db = DB::conn();
+        $rows = $db->rows("SELECT * FROM user WHERE id = ?", array($id));
+        
+        foreach ($rows as $row) {
+            $user[] = new self($row);
+            }
+        return $user;
+        }
+
+    //Gets user's Data
+    public static function getData($user_id){
+    $db = DB::conn();
+    $row = $db->row("SELECT * FROM user
+        WHERE id = ?", array($user_id));
+    if (!$row) {
+    throw new RecordNotFoundException('no record found');
+    }
+    return new self($row);
+    }
```

Comment:
thanks po, i already deleted getAll();
regarding with get() and getData, I tried deleting get(), and used getData() instead but i'm getting an error:
Catchable fatal error: Argument 1 passed to Model::__construct() must be an array, boolean given( ...)

so i changed return new self $row into return $row .
but now, i cant call the addFollowing() function from the controller.

I retained the get() function, is that okay po?